src/ts/localization/i18nApi.ts

```diff
@@ -0,0 +1,161 @@
+import { defaultBitmovinPlayerUiTranslations } from './i18n';
+
+interface StaticVocabulary {
+  'settings': string;
+  'settings.video.quality': string;
+  'settings.audio.quality': string;
+  'settings.audio.track': string;
+  'settings.audio.mute': string;
+  'settings.window.color': string;
+  'settings.window.opacity': string;
+  'settings.subtitles': string;
+  'settings.subtitles.font.color': string;
+  'settings.subtitles.font.opacity': string;
+  'settings.subtitles.background.color': string;
+  'settings.subtitles.background.opacity': string;
+  'settings.fullscreen': string;
+  'settings.speed': string;
+  'settings.playPause': string;
+  'settings.open': string;
+  'labels.pictureInPicture': string;
+  'labels.appleAirplay': string;
+  'labels.googleCast': string;
+  'labels.vr': string;
+  'labels.off': string;
+  'labels.auto': string;
+  'labels.back': string;
+  'labels.reset': string;
+  'labels.replay': string;
+  'labels.normal': string;
+  'labels.default': string;
+  'colors.white': string;
+  'colors.black': string;
+  'colors.red': string;
+  'colors.green': string;
+  'colors.blue': string;
+  'colors.cyan': string;
+  'colors.yellow': string;
+  'colors.magenta': string;
+  'opacity.100': string;
+  'opacity.75': string;
+  'opacity.50': string;
+  'opacity.25': string;
+  'opacity.0': string;
+  'settings.subtitles.font.size': string;
+  'settings.subtitles.font.size.50': string;
+  'settings.subtitles.font.size.75': string;
+  'settings.subtitles.font.size.100': string;
+  'settings.subtitles.font.size.150': string;
+  'settings.subtitles.font.size.200': string;
+  'settings.subtitles.font.size.300': string;
+  'settings.subtitles.font.size.400': string;
+  'settings.subtitles.characterEdge': string;
+  'settings.subtitles.characterEdge.raised': string;
+  'settings.subtitles.characterEdge.depressed': string;
+  'settings.subtitles.characterEdge.uniform': string;
+  'settings.subtitles.characterEdge.dropshadowed': string;
+  'settings.subtitles.font.family': string;
+  'settings.subtitles.font.family.monospacedserif': string;
+  'settings.subtitles.font.family.proportionalserif': string;
+  'settings.subtitles.font.family.monospacedsansserif': string;
+  'settings.subtitles.font.family.proportionalsansserif': string;
+  'settings.subtitles.font.family.casual': string;
+  'settings.subtitles.font.family.cursive': string;
+  'settings.subtitles.font.family.smallcapital': string;
+  'messages.ads.remainingTime': string;
+}
+
+export type Vocabulary<V> =  V & Partial<StaticVocabulary>;
+type StringKeyMap = {[key: string]: string};
+
+export interface BitmovinPlayerUiTranslations {
+  [key: string]: Vocabulary<StringKeyMap>;
+}
+
+export interface LocalizationConfig {
+  language: string;
```

Comment:
might be a good idea. We should have a discussion about it tough because we currently have only 'en' and 'de' and as far as I know we are not planning to provide additional languages.
another possibility would be to have `language: 'de' | 'en' | string` as typedef. and user can add their language or can see what we currently have?

Reply:
True we don't have much 🤔 An enum might be an overkill.